go.mod

```diff
@@ -1,37 +1,37 @@
 module github.com/fnproject/cli
 
 require (
-	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/coreos/go-semver v0.2.0
-	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/fatih/color v0.0.0-20170926111411-5df930a27be2
-	github.com/fnproject/fn_go v0.5.0
+	github.com/fnproject/fn_go v0.5.2-0.20200306235942-b9463d5f4f6b
```

Comment:
Revendor to use latest fn_go with changes to add CS provider.

Reply:
Done.